### PR TITLE
Added RPR_InsertMedia as Project.import_media() and SetMediaItemSelected as Project.select_item()

### PR DESCRIPTION
**Pull request by makingpippop**
Hi!

I've added the method to import a file in a REAPER project.
To use it simply call: 

`project.import_media('path/to/file')`

REAPER will create a new track and place the imported `Item` at the beginning of the project [00:00].
If the file is not found or invalid it will raise an Exception

You can also use the argument
 `addToSelectedTrack : bool [optional]` 
To import the media to the selected track in the project.
If no track is selected, it will raise an `IndexError` 

 `setToCursorPosition : bool [optional]` 
To set the imported Item's position to the cursor position

I also added a tool called `file_handler.validate_path('path/to/file/or/folder',ext=None, isFolder=False)` 
Basically this method executes some validation on the path and raises for multiple reasons. 
1. The file or folder doesn't exists (The path is relative to the project location [`reapy.Project().path`])
2. The file type is not supported by REAPER
3. The file type does not match the extension passed as an optional argument "`ext`".

### CODE REVIEW

**Comment by dagelabi** [comment]
Sorry! I totally forgot that I've also added `project.select_item()` which allow to select a specific item in REAPER.

By default, the item will be added to the current selection. By using the argument `makeUnique=True` the current selection will be removed and only the specified Item will be selected.

File: reapy/core/project/project.py [hunk 1]
```diff
@@ -12,1240 +13,1303 @@
 
 class Project(ReapyObject):
 
-    """REAPER project."""
+	"""REAPER project."""
```
Comment (Levitanus) [inline reapy/core/project/project.py:16]: I feel some of code formatters worked here.... 
I have also such problems, while modifying reapy. But, can You try to leave some way only lines that are changed?

Usually I use just an simple pep-8 formatter in very gentle mode.

File: reapy/core/project/project.py [hunk 2]
```diff
@@ -12,1240 +13,1303 @@
...
+	def import_media(self, filepath, addToSelectedTrack=False, setToCursorPosition=False):
+		"""
+		Imports a file and place the media on a track.
+
+		Parameters
+		----------
+		filepath : str
+			Filepath to the file to import (relative to the REAPER project)
+		addToSelectedTrack : bool, optional
+			Instead of creating a new track, a new track will be created.
```
Comment (Levitanus) [inline reapy/core/project/project.py:161]: > The selected one will be used?

))

**Review by Levitanus — COMMENTED** [review]
I'm afraid, if still item object is needed — it's quite more flexible to make everything on the item side.
More than, there are some methods like `make_only_selected_iteem` or something like these. I don't feel such project API is suitable here...

File: reapy/core/project/project.py (hunk 2 above)
Comment (dagelabi) [reply reapy/core/project/project.py:161]: Oups! thanks!

File: reapy/core/project/project.py (hunk 1 above)
Comment (dagelabi) [reply reapy/core/project/project.py:16]: Yeah, I didn't think before converting spaces to tabs... What would be the best way to fix this mistake?

**Comment by Levitanus** [comment]
I'm not a valid reviewer here, so these are just suggestions as from contributor, But maybe they help)) Thanks!

File: reapy/core/project/project.py (hunk 1 above)
Comment (dagelabi) [reply reapy/core/project/project.py]: Fixed! 0d10453

**Comment by komjum6** [comment]
Can't figure out why this isn't merged

**Comment by drunkenQCat** [comment]
> Can't figure out why this isn't merged

Maybe the indent of his code is not matched to original repo.

**Comment by Levitanus** [comment]
Repository is not maintained. You always can use reapy-boost from my repository. And I try to approve all PR's, which do not break the repo. But I also almost forgot of this project, as stated to write extensions in rust, and, even added a lot of reapy philosophy there.

**Comment by drunkenQCat** [comment]
> Repository is not maintained. You always can use reapy-boost from my repository. And I try to approve all PR's, which do not break the repo. But I also almost forgot of this project, as stated to write extensions in rust, and, even added a lot of reapy philosophy there.

Thanks a lot for your reply,  And I am happy to use see your works.  Actually, I am a new one to reaper programming,  and I am going to spend time on translate reapy to c# version.  I will read your code for further inspiration, thank you.
